includes/admin/discounts/edit-discount.php

```diff
@@ -127,186 +126,162 @@
-						) );
-
-						echo esc_html( ' (' . edd_get_timezone_abbr() . ')' );
-						?>
-                        <p class="description"><?php _e( 'Pick the date this discount will start on. Leave blank for no start date.', 'easy-digital-downloads' ); ?></p>
-                    </td>
-                </tr>
-
-                <?php do_action( 'edd_edit_discount_form_before_expiration', $discount->id, $discount ); ?>
-
-                <tr>
-                    <th scope="row" valign="top">
-                        <label for="edd-expiration"><?php _e( 'Expiration date', 'easy-digital-downloads' ); ?></label>
-                    </th>
-                    <td>
-                        <input name="end_date" id="edd-expiration" type="text" value="<?php echo esc_attr( false !== $discount->end_date ? $end_date : '' ); ?>"  class="edd_datepicker" data-format="<?php echo esc_attr( edd_get_date_picker_format() ); ?>" placeholder="<?php echo esc_attr( edd_get_date_picker_format() ); ?>" />
-						<?php
-						echo EDD()->html->select( array(
-							'name'             => 'end_date_hour',
-							'options'          => $hours,
-							'selected'         => false !== $discount->end_date ? $end_hour : '23',
-							'chosen'           => true,
-							'class'            => 'edd-time',
-							'show_option_none' => false,
-							'show_option_all'  => false
-						) );
-						?>
+
+						<label class="screen-reader-text" for="start-date-minute">
+							<?php esc_html_e( 'Start Date Minute', 'easy-digital-downloads' ); ?>
+						</label>
+						<input type="number" min="0" max="59" step="1" name="start_date_minute" id="start-date-minute" value="<?php echo esc_attr( false !== $discount->start_date ? $start_minute : '' ); ?>" placeholder="00" />
+
+						<?php echo esc_html( ' (' . edd_get_timezone_abbr() . ')' ); ?>
+						<p class="description"><?php _e( 'Pick the date and time this discount will start on. Leave blank for no start date.', 'easy-digital-downloads' ); ?></p>
+					</td>
+				</tr>
+
+				<?php do_action( 'edd_edit_discount_form_before_expiration', $discount->id, $discount ); ?>
+
+				<tr>
+					<th scope="row" valign="top">
+						<label for="edd-expiration"><?php _e( 'Expiration date', 'easy-digital-downloads' ); ?></label>
+					</th>
+					<td class="edd-discount-datetime">
+						<input name="end_date" id="edd-expiration" type="text" value="<?php echo esc_attr( false !== $discount->end_date ? $end_date : '' ); ?>"  class="edd_datepicker" data-format="<?php echo esc_attr( edd_get_date_picker_format() ); ?>" placeholder="<?php echo esc_attr( edd_get_date_picker_format() ); ?>" />
+
+						<label class="screen-reader-text" for="end-date-hour">
+							<?php esc_html_e( 'Expiration Date Hour', 'easy-digital-downloads' ); ?>
+						</label>
+						<input type="number" min="0" max="24" step="1" name="end_date_hour" id="end-date-hour" value="<?php echo esc_attr( false !== $discount->end_date ? $end_hour : '' ); ?>" placeholder="00" />
```

Comment:
```suggestion
						<input type="number" min="0" max="24" step="1" name="end_date_hour" id="end-date-hour" value="<?php echo esc_attr( false !== $discount->end_date ? $end_hour : '' ); ?>" placeholder="23" />
```

To match the same placeholder value as on the `Add New Discount` page.